optimization/ht_functions.py

```diff
@@ -1,12 +1,128 @@
-# Other Imports
 import math
-import numpy as np
-import operator
-import sys
-from scipy.optimize import minimize, minimize_scalar
-# Import physical constants
-from physical_constants import const, FlowProperties
+from scipy.optimize import minimize_scalar
+# import physical properties
+import physical_properties as pp
 
+def pipeflow_turbulent(Re, Pr, LD, relrough):
+    """Turbulent pipeflow correlation from EES. This function proiveds a nusselt
+    number and friction factor for turbulent flow in a pipe.
+
+    Arguments:
+    ----------
+        Re (float): flow Reynold's number [-]
+        Pr (float): flow Prandtl number [-]
+        LD (float): length over diameter [-]
+        relrough (float) : relative roughness [-]
+    
+    Returns:
+    --------
+        Nusselt_L (float): nusselt number
+        f (float): friction factor
+    """
```

Comment:
Maybe you should add some test there to ensure this is not used on a laminar case ?
(as it seems to only depends on `Re`)